build/lib/electron.ts

```diff
@@ -91,7 +91,7 @@ function darwinBundleDocumentTypes(types: { [name: string]: string | string[] },
 }
 
 export const config = {
-	version: product.electronRepository ? '19.1.11' : util.getElectronVersion(),
+	version: product.electronRepository ? '22.10.0' : util.getElectronVersion(),
```

Comment:
I will update this to an official version once the PGO related builds are available in a follow-up PR. For now, lets merge with the custom version to get insiders coverage.